builder/assignmentEnvImageBuilder.go

```diff
@@ -69,7 +78,10 @@ func withAssgnEnvConfig(assignCfgs *configurations.AssignmentEnvConfig) assignme
 	}
 }
 
-func (assgnEnv *assignmentEnvironment) verifyAndWriteInstructions() error {
+// verifyAndWriteInstructions checks whether the provided language image is already
+// present in docker hub and accordingly writes the Dockerfile from either base image
+// or from the existing language image. It returns any error encountered.
```

Comment:
don't have to mention that it returns an error.

Reply:
Okay